src/Smalot/PdfParser/Page.php

```diff
@@ -210,7 +210,10 @@ public function getText(self $page = null): string
                 $contents = new PDFObject($this->document, $header, $new_content, $this->config);
             }
 
-            return $contents->getText($this);
+            $contentsText = $contents->getText($this);
+            PDFObject::$recursionStack = [];
```

Comment:
Please add a short comment about those two lines.

Reply:
Here in the code or above the integration test?

Reply:
In the code above the two lines I marked please, so developers get an idea why this code is used.

Reply:
I've added a comment.